### ISSUE TITLE
Training parameters for training EG3D on the masked FFHQ dataset

### ISSUE THREAD

**Issue by FANG-HJ** [tobias-kirschstein/gghead#17]
Thanks for the impressive work!

In your paper, I noticed that you utilized MODNet to generate a masked version of FFHQ with a white-pixel background. I was wondering if you trained EG3D using the dataset you shared a few weeks ago. Could you please provide more details about the training parameters you configured for this process? Additionally, if you fine-tuned the model, could you elaborate on how the training parameters were adjusted.

**Comment by tobias-kirschstein** [comment]
Hi,

we trained EG3D on the masked dataset with the default hyperparameters from EG3D.
The only thing we changed besides the dataset is to ensure that the rendering background color for EG3D is white to encourage rays hitting background pixels to be empty.
